lib/rubygems/errors.rb

```diff
@@ -92,12 +92,18 @@ def initialize(target, conflicts)
       @conflicts = conflicts
       @name      = target.name
 
+
```

Comment:
@bronzdoc looks like this extra newline [caused Rubocop to be unhappy](https://travis-ci.org/rubygems/rubygems/jobs/552446705#L316), which failed the entire build.